src/pages/auth/Auth.tsx

```diff
@@ -32,12 +32,21 @@ function Auth() {
   const [profileImgIsUp, setProfileImgIsUp] = useState(false);
   const socket = getSocket();
 
+  const errorListener = (res: { status: string; detail: string }) => {
+    if (res.status === "error") {
+      alert("이미 접속 중 입니다.");
+      window.open('about:blank','_parent')?.parent.close();
```

Comment:
```suggestion
      window.open('about:blank','_parent')?.parent.close();
```

이 코드 작동이 빈 페이지 열어주고 기존 창 (parent) 닫아주는게 맞나요???

Reply:
로그인 화면에서 테스트 하시려면 
- 크롬탭1 에서 로그인(id: ho), 시크릿탭에서 로그인(id: ho)시도 -> '이미 접속 중 인 아이디 입니다.'
- 크롬탭1 에서 로그인(id: ho), 크롬탭2 에서 사이트(localhost) 접속 -> '이미 접속 중 입니다.' alert
이런식입니다.

네 맞습니다. 빈 페이지 열어주고 기존 창 닫는거 맞아요 빈 페이지 안 띄우고 싶었는데 현재 크롬 브라우저에서 보안상 이유로 막았다고 하네요 좀 더 좋은 방법 찾으면 그때 또 개선해볼게요!